testCPUID-all/testCPUID.cpp

```diff
@@ -411,18 +446,20 @@ void experiment(LPTHREAD_START_ROUTINE proc)
 
     if (g_worker_core >= 0)
     {
+        printf("Worker core: ");
         SetThreadAffinity(g_hThread, g_worker_core);
     }
 
     if (g_mainthread_core >= 0)
     {
+        printf("Main core: ");
         SetThreadAffinity(GetCurrentThread(), g_mainthread_core);
     }
 
     ResumeThread(g_hThread);
 
     // sleep for 10s
-    Sleep((DWORD)10 * 1000);
+    Sleep((DWORD)20 * 1000);
```

Comment:
why change it to 20s?

Reply:
Just so that I can take profile measurements for longer duration. I will revert it.

Reply:
ahh, I was just a little surprised that 10s wasn't enough for something like. I don't have any objections to changing it to 20s.